test/unit/alignment/pairwise/global_affine_unbanded_test.cpp

```diff
@@ -0,0 +1,214 @@
+
+    std::vector database = fixture.sequence1;
+    std::vector query = fixture.sequence2;
+
+    // TODO Make test work with ranges.
+    auto alignment = align_pairwise(std::pair{database, query}, align_cfg);
+
+    EXPECT_EQ((*std::ranges::begin(alignment)).get_score(), fixture.score);
+}
+
+// TYPED_TEST_P(global_affine_unbanded, score_matrix)
+// {
+//     using word_type = typename TypeParam::word_type;
+//     auto const & fixture = this->fixture();
+//     auto align_cfg = fixture.config;
+//
+//     std::vector database = fixture.sequence1;
+//     std::vector query = fixture.sequence2;
+//
+//     auto alignment = edit_distance<word_type>(database, query, align_cfg);
+//     auto score_matrix = alignment.score_matrix();
+//
+//     EXPECT_EQ(score_matrix.cols(), database.size()+1);
+//     EXPECT_EQ(score_matrix.rows(), query.size()+1);
+//     EXPECT_EQ(score_matrix, fixture.score_matrix);
+//     EXPECT_EQ(alignment.score(), fixture.score);
+// }
+//
+// TYPED_TEST_P(global_affine_unbanded, trace_matrix)
+// {
+//     using word_type = typename TypeParam::word_type;
+//     auto const & fixture = this->fixture();
+//     auto align_cfg = fixture.config;
+//
+//     std::vector database = fixture.sequence1;
+//     std::vector query = fixture.sequence2;
+//
+//     auto alignment = edit_distance<word_type>(database, query, align_cfg);
+//     auto trace_matrix = alignment.trace_matrix();
+//     auto begin_coordinate = alignment.begin_coordinate();
+//     auto end_coordinate = alignment.end_coordinate();
+//
+//     EXPECT_EQ(trace_matrix.cols(), database.size()+1);
+//     EXPECT_EQ(trace_matrix.rows(), query.size()+1);
+//     EXPECT_EQ(begin_coordinate.seq1_pos, fixture.begin_coordinate.seq1_pos);
+//     EXPECT_EQ(begin_coordinate.seq2_pos, fixture.begin_coordinate.seq2_pos);
+//     EXPECT_EQ(end_coordinate.seq1_pos, fixture.end_coordinate.seq1_pos);
+//     EXPECT_EQ(end_coordinate.seq2_pos, fixture.end_coordinate.seq2_pos);
+//     EXPECT_EQ(trace_matrix, fixture.trace_matrix);
+//     EXPECT_EQ(alignment.score(), fixture.score);
+//
+//     auto && [gapped_database, gapped_query] = alignment.trace();
+//     EXPECT_EQ(std::string{gapped_database | view::to_char}, fixture.gapped_sequence1);
+//     EXPECT_EQ(std::string{gapped_query | view::to_char}, fixture.gapped_sequence2);
+// }
+//
+// TYPED_TEST_P(global_affine_unbanded, trace)
+// {
+//     using word_type = typename TypeParam::word_type;
+//     auto const & fixture = this->fixture();
+//     auto align_cfg = fixture.config;
+//
+//     std::vector database = fixture.sequence1;
+//     std::vector query = fixture.sequence2;
+//
+//     auto alignment = edit_distance<word_type>(database, query, align_cfg);
+//
+//     auto && [gapped_database, gapped_query] = alignment.trace();
+//     EXPECT_EQ(std::string{gapped_database | view::to_char}, fixture.gapped_sequence1);
+//     EXPECT_EQ(std::string{gapped_query | view::to_char}, fixture.gapped_sequence2);
+// }
```

Comment:
...?

Reply:
But why commit dead code? If the other PRs change these lines anyway, why can't they add the code in the first place?

Reply:
Well, well, I'll see that I remove it.